test/error_tracker/ignorer_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule ErrorTracker.IgnorerTest do
+  use ErrorTracker.Test.Case
+
+  setup_all do
+    Application.put_env(:error_tracker, :ignorer, ErrorTracker.EveryErrorIgnorer)
```

Comment:
Do the env restore itself after every execution? I'm not sure.

Reply:
I (incorrectly) thought so. 

I've updated the test case to:
1. Use `setup` instead of `setup_all`
2. Use the `on_exit` callback to restore the previous environment value after finishing the test
3. Add another test for the case without an ignorer, which would have allowed us to catch the error that you spotted in your previous commit.